src/ui/src/tabnamemapping.cpp

```diff
@@ -84,7 +84,8 @@ void TabNameMapping::retrieveFromStorage( QSettings& settings )
     if ( settings.contains( "TabNameMapping/version" ) ) {
         settings.beginGroup( "TabNameMapping" );
         if ( settings.value( "version" ).toInt() == TABNAMEMAPPING_VERSION ) {
-            int size = settings.beginReadArray( "tabNames" );
+            const int size = settings.beginReadArray( "tabNames" );
```

Comment:
Another settings place. Let's remove reserve

Reply:
fix it https://github.com/variar/klogg/commit/a4f9cad9eceff87a667f1c6cec939f383153a6e3